homework3.py

```diff
@@ -0,0 +1,86 @@
+import functools
+
+
+def divided_by_one_hundred(func):
+    @functools.wraps(func)
+    def inner(*args, **kwargs):
+        res = func(*args, **kwargs)
+        if res % 100 == 0:
+            print("We are OK!")
+        else:
+            print(f"Bad news guys, we got {res % 100}")
+        return res
+    return inner
```

Comment:
If I am not mistaken, the task was to check if 100 can be divided by the number without remainder of the division